tests/foreman/cli/test_syncplan.py

```diff
@@ -6,7 +6,7 @@
 from fauxfactory import gen_string
 from robottelo.cli.factory import CLIFactoryError, make_org, make_sync_plan
 from robottelo.cli.syncplan import SyncPlan
-from robottelo.common.decorators import data
+from robottelo.common.decorators import (data, skip_if_bug_open)
```

Comment:
Nitpick: Drop the parentheses. Parentheses are great when imports need to be spread over multiple lines, but are unnecessary in this context.